### PR TITLE
(Feature) - Manage Crowdsale - Able to add new addresses to Whitelist

### PR DESCRIPTION
**Pull request by fernandomg**
This is the last step towards the completion of #323 

- Allows addition of addresses to the whitelist.
  - If a Crowdsale has 3-tiers, adding an address to the 1st tier, will generate 3 Tx, as the 2nd and 3rd tiers has to inherit the address as well. Is this the behavior expected?
- Depends on #404 **So it's not safe to merge it to `master` before #404**

### CODE REVIEW

**Comment by vbaranov** [comment]
@fernandomg please, resolve conflicts.

**Comment by fernandomg** [comment]
@vbaranov done

**Comment by vbaranov** [comment]
>registry contract should be in the mainnet too

0x445b4c58113C784640a978cfE39b0065118B91A5

File: .env.example [hunk 1]
```diff
@@ -1 +1 @@
-REACT_APP_REGISTRY_ADDRESS='{"3":"0xD4543Fe1ded1637768516A3Ef35614F91F715370","4":"0x16C25e1139fC96E8705A59E7a4F132d11DfFeE4E","42":"0x3d9EE1B10a002B9113e5dAeF8F19067503d6162A","12648430":"0xe78a0f7e598cc8b0bb87894b0f60dd2a88d6a8ab","77":"0x058faf0b1248a6de96ef51696564fe5d64b5e231","99":"0x41D420ED515E51A4023fBa2945a46186274181d6"}'
+REACT_APP_REGISTRY_ADDRESS='{"1":"0x445b4c58113C784640a978cfE39b0065118B91A5",""3":"0xD4543Fe1ded1637768516A3Ef35614F91F715370","4":"0x16C25e1139fC96E8705A59E7a4F132d11DfFeE4E","42":"0x3d9EE1B10a002B9113e5dAeF8F19067503d6162A","12648430":"0xe78a0f7e598cc8b0bb87894b0f60dd2a88d6a8ab","77":"0x058faf0b1248a6de96ef51696564fe5d64b5e231","99":"0x41D420ED515E51A4023fBa2945a46186274181d6"}'
```
Comment (vbaranov) [inline .env.example]: typo: `""3"`

Comment (fernandomg) [reply .env.example]: oh, thanks!

**Comment by fernandomg** [comment]
@vbaranov it's ok if I change the alignment for the whitelist form too? so the items are properly aligned with the inputs? (it will affect too `stepThree`)

currently (from `stepThree`)
image

final result (from `manage`, in step three it looks ok too)
image

**Comment by vbaranov** [comment]
@fernandomg yes, final result looks better!

**Comment by fernandomg** [comment]
> a) clear corresponding mobx stores at the start of the wizard process

I'll go with this option, as I think its the most expected/natural behavior

**Pull request merged**